src/base_home.cpp

```diff
@@ -0,0 +1,94 @@
+#include "base_home.h"
+#include "submap.h"
+#include <unordered_map>
+#include "mapdata.h"
+#include "map.h"
+
+base_home::base(submap &base_map,const tripoint &coreloc){
+    base::define_base_area(base_map, coreloc);
+    base_map.spawns.clear();//remove spawns so nothing apears in base just outside it.
+    base_level = 1;
+    food_ration_lv = ammo_ration_lv = med_ration_lv = 0; //no rationing to start
```

Comment:
Members should be initialized in the member initialization list, not assigned in the body of the constructor.

Reply:
hmm... I probably did this in reference to it being done another header file.  The idea was to default the values incase there was an error somewhere.